dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -117,6 +120,30 @@ public void testWorkflowMigration() throws Exception {
             assertEquals("Should not be able to refresh a dockstore.yml workflow.", HttpStatus.SC_BAD_REQUEST, ex.getCode());
         }
 
+        // Should be able to refresh a legacy version
```

Comment:
Could use a test for refresh version on a frozen version

Reply:
Actually this is hard to test, since we would be testing that a frozen version is not removed when the corresponding Git repo branch is deleted. I don't think our current testing framework supports this.

Reply:
That would be hard.  How about testing that a frozen version is not updated when the corresponding Git repo branch is newer?

Reply:
Wouldn't that also require making a push to the Git repo?

Reply:
What if I do a refresh and check the dbupdatetime?

Reply:
Was more thinking that you could start the database off with different content, freeze it, do a fake call to the new endpoint, and just verify that the new endpoint respects the frozen status and doesn't update to whats in GitHub